plugins/UsersManager/lang/en.json

```diff
@@ -4,150 +4,150 @@
         "CreateNewToken": "Create new token",
-        "AnonymousAccessConfirmation": "You are about to grant the anonymous user the 'view' access to this website. This means your analytics reports and your visitors information will be publicly viewable by anyone even without a login. Are you sure you want to proceed?",
+        "AnonymousAccessConfirmation": "Grant the anonymous user the 'view' access to this website? This means your analytics reports and your visitor will be publicly viewable by anyone even without a login.",
         "AnonymousUser": "Anonymous user",
-        "AnonymousUserHasViewAccess": "Note: the %1$s user has %2$s access to this website.",
-        "AnonymousUserHasViewAccess2": "Your analytics reports and your visitors information are publicly viewable.",
+        "AnonymousUserHasViewAccess": "Note: The %1$s user has %2$s access to this website.",
+        "AnonymousUserHasViewAccess2": "Your analytics reports and your visitor info is publicly viewable.",
         "ApplyToAllWebsites": "Apply to all websites",
-        "ChangeAllConfirm": "Are you sure you want to give '%s' access to all the websites?",
+        "ChangeAllConfirm": "Give '%s' access to all the websites?",
         "ClickHereToDeleteTheCookie": "Click here to delete the cookie and have Matomo track your visits",
-        "ClickHereToSetTheCookieOnDomain": "Click here to set a cookie that will exclude your visits on websites tracked by Matomo on %s",
-        "ConfirmGrantSuperUserAccess": "Do you really want to grant '%s' Super User access? Warning: the user will have access to all websites and will be able to perform administrative tasks.",
-        "ConfirmProhibitMySuperUserAccess": "%s, do you really want to remove your own Super User access? You will lose all permissions and access to all websites and will be logged out from Matomo.",
-        "ConfirmProhibitOtherUsersSuperUserAccess": "Do you really want to remove Super User access from '%s'? The user will lose all permissions and access to all websites. Make sure to give access to needed websites afterwards if necessary.",
-        "DeleteConfirm": "Are you sure you want to delete the user %s?",
-        "Email": "Email",
+        "ClickHereToSetTheCookieOnDomain": "Click here to set a cookie to exclude your visits on websites tracked by Matomo on %s",
+        "ConfirmGrantSuperUserAccess": "Grant '%s' superuser access? Warning: The user will have access to all websites and will be able to perform administrative tasks.",
+        "ConfirmProhibitMySuperUserAccess": "%s, do you really want to remove your own superuser access? You will lose all permissions and access to all websites, and will be logged out of Matomo.",
+        "ConfirmProhibitOtherUsersSuperUserAccess": "Revoke superuser access from '%s'? The user will lose all permissions and access to all websites. Make sure you grant access to needed websites afterwards if necessary.",
+        "DeleteConfirm": "Delete the '%s' user?",
```

Comment:
The placeholder will be filled with the users login. Not sure if the order needs to be changed in that case.